wpimath/src/main/java/edu/wpi/first/math/interpolation/InterpolatingDoubleTreeMap.java

```diff
@@ -13,4 +15,19 @@ public class InterpolatingDoubleTreeMap extends InterpolatingTreeMap<Double, Dou
   public InterpolatingDoubleTreeMap() {
     super(InverseInterpolator.forDouble(), Interpolator.forDouble());
   }
+
+  /**
+   * Creates an {@link InterpolatingDoubleTreeMap} from a list of entries.
```

Comment:
nit: Technically a `List` isn't provided.
```suggestion
   * Creates an {@link InterpolatingDoubleTreeMap} from the given entries.
```